packages/lib-user/package.json

```diff
@@ -24,30 +24,59 @@
     "build:cjs": "babel ./src/ --out-dir ./dist/cjs --copy-files --no-copy-ignored",
     "build:es6": "BABEL_ENV=es6 babel ./src/ --out-dir ./dist/esm --copy-files --no-copy-ignored",
     "dev": "BABEL_ENV=es6 webpack serve --config webpack.dev.js",
+    "eslint": "eslint ./src --fix | snazzy",
     "prepare": "yarn build",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "BABEL_ENV=es6 mocha --loader=@node-loader/babel --config ./test/.mocharc.json \"./src/**/*.spec.js\"",
```

Comment:
While getting mocha to run tests in the lib-user ESM package, I ran into similar issues noted in #3779 . The script here works, but there might be additional issues as more/actual tests are written. It's been discussed in Slack of trying Jest or an alternative to mocha.

Reply:
Noting here just for Github visibility that I think it's fine to leave out mocha setup in this PR if you'd like to experiment with other testing solutions in lib-user (time allowing). Or if easier to move ahead with mocha that works too!

Reply:
I think I'd like to leave mocha in for now, but I'm thinking I add one (small) actual component with story and tests (with mocha), import the component into app-root to confirm works as expected, then attempt to refactor mocha with jest. Always open to suggestion, but that's my tentative plan for now.